config.go

```diff
@@ -114,6 +115,13 @@ func unmarshalTypedConfig(cfg map[string]interface{}) error {
 
 // apply applies the configuration.
 func (c *srcfileConfig) apply() error {
+	// KLUDGE: determine whether we're in the stdlib and if so, set GOROOT to "." before applying config.
+	// This is necessary for the stdlib unit names to be correct.
+	cloneURL, _ := exec.Command("git", "config", "--get", "remote.origin.url").CombinedOutput()
+	if strings.TrimSpace(string(cloneURL)) == "https://github.com/golang/go" && c.GOROOT == "" {
```

Comment:
Would the clone URL be `github.com` during a build, or would it try to import the repo from the local Sourcegraph server? I think repo dependencies are fetched directly from github.com via `go get`, but when building the stdlib itself, the clone URL would be the local server's URL (eg https://sourcegraph.com/github.com/golang/go).